src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_BreakingChangesReport.cshtml

```diff
@@ -18,8 +18,8 @@
 
 <h2 id="@LocalizedStrings.CompatibilityPageTitle" tabindex="0">@LocalizedStrings.CompatibilityPageTitle</h2>
 <div id="summary-compat-byassembly">
-    <h3 class="compat-subheader">@LocalizedStrings.PerAssemblyOverview</h3>
-    <table>
+    <h3 class="compat-subheader" id="compat-byassembly-subheader">@LocalizedStrings.PerAssemblyOverview</h3>
```

Comment:
With this change, will @LocalizedStrings.PerAssemblyOverview be read two times, one as h3 and one as table. Also per https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/label-labelledby/, aria-label and aria-labelledby has some undesirable side effects. What if remove <h3 class="compat-subheader">, and add caption with @LocalizedStrings.PerAssemblyOverview as value to the table?

Reply:
`<caption>` works but Narrator reads it twice in Edge, once in Firefox. I agree it's preferable semantically, so I'll push a commit labeling the tables with `<caption>`.